cs/ccxt/ws/Future.cs

```diff
@@ -38,7 +38,8 @@ public void resolve(object data = null)
 
         public void reject(object data)
         {
-            this.tcs.SetException(new Exception(data.ToString()));
+            var exception = (data is Exception) ? data as System.Exception : new Exception(data.ToString()); 
```

Comment:
even with this PR the user won't get the "true" exception
![image](https://github.com/user-attachments/assets/1432a64a-9cf6-46d4-a4b4-471bb4efaef3)

Reply:
yes but what else can be done? this way is one step better (the maximum that can be done) and users can access `.InnerException`